mlir-sycl/lib/Conversion/SYCLToLLVM/SYCLToLLVM.cpp

```diff
@@ -528,6 +528,8 @@ class ConstructorPattern final
   }
 };
 
+// TODO: Push fix when testing is possible.
+#if 0
```

Comment:
I guess you need to fix this code?

Reply:
Why is this code commented out ?

Reply:
Answered above

Reply:
Given we have many outstanding PRs we can let this go in this PR, then add the XFAIL test back along with a code fix in a future PR.